qa/tasks/cephfs/test_data_scan.py

```diff
@@ -494,8 +494,8 @@ def test_fragmented_injection(self):
         split_size = 100 * file_count
 
         # Make sure and disable dirfrag auto merging and splitting
-        self.fs.mon_manager.run_cluster_cmd(args='config set mds mds_bal_merge_size 0')
-        self.fs.mon_manager.run_cluster_cmd(args=f'config set mds mds_bal_split_size {split_size}')
+        self.config_set('mds', 'mds_bal_merge_size', 0)
```

Comment:
Is this the general practice to make the suggestion into different commit ? I don't have strong opinion here but  I generally  find it cluttered to have the suggestion a separate commit.

Reply:
I used the UI to merge the suggestion and that creates an additional commit. I'll squash it before merging.

Reply:
https://github.com/ceph/ceph/pull/54590#issuecomment-1820543576